README.md

```diff
@@ -1,2 +1,29 @@
 # ariitk_template
 Template repository for ease of setting up a new project
+
+
+# Installation and Usage
+
+run `sudo ./setup.sh` to install this package.
+
+For making meta_pkg , go into the src folder of your workspace-
+run `ariitk_create_metapkg`   
+change the template file according to your need and press `:x` to save changes.
+
+For making simple_pkg , go into the src folder of your workspace-  
+run  `ariitk_create_pkg <pkg_name> <node_name> <dependencies>`   
+
+For creating a dependencies.rosinstall, go into the package.  
+run ```ariitk_rosinstall``` : vim editor will open a file with ```repositories=""```   
+Fill in the common name of the repositories you want. Some will already be available by default, for others you will get a prompt to set the github link.   
+The list of common repos and their links is available inside ariitk_rosinstall.
+
+# Without installation
+
+
+Use command `wget https://tinyurl.com/ariitk && chmod +x ariitk && ./ariitk && rm -rf ariitk` to create a meta package and setup git remote accordingly.You do not need to clone this package to run this command.
```

Comment:
did you test the shorter one i gave you?

Reply:
@ashwin2802 
I tried the shorter version. So it ran and saved the script in `ariitk`. After that I had to run ```bash ariitk``` in order to create the packages. And it's still saved in the home directory.
The longer command bypasses this step altogether. So what we can do is give the user both the commands (incase they want to modify the script to their liking). Sound fine?

Reply:
problem was that our script is interactive , so the command is not working 

`bash <(wget -qO -  https://tinyurl.com/ariitk )` 
this is the right command .

i dont know why source command doesn't work well with shebang /bin/sh

so , `sh <(wget -qO -  https://tinyurl.com/ariitk)` when you use this command , you will get an error.